crates/tabby-common/src/api/accelerator.rs

```diff
@@ -0,0 +1,21 @@
+use juniper::{GraphQLEnum, GraphQLObject};
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+#[derive(Serialize, Deserialize, GraphQLEnum, ToSchema, PartialEq, Clone, Debug)]
+pub enum DeviceType {
+    Cuda,
+    Rocm,
+}
+
+#[derive(Serialize, Deserialize, GraphQLObject, ToSchema, Clone, Debug)]
+pub struct Accelerator {
+    /// Universally unique ID of the accelerator, if available
+    pub uuid: Option<String>,
```

Comment:
The biggest issue now is whether we can prevent this field from being part of the telemetry. If not, it definitely has to be removed, as it's basically almost equivalent to a serial number. It's nice for tracking a GPU in the accelerators, but it's a huge issue if that data leaves your control.